core/spec/models/spree/order_cancellations_spec.rb

```diff
@@ -155,7 +155,7 @@
         inventory_unit_2.line_item.reload # UnitCancel#compute_amount needs updated amounts
         order.cancellations.short_ship([inventory_unit_2])
         line_item.reload
-        expect(line_item.adjustments.non_tax.sum(:amount)).to eq(-1.67)
+        expect(line_item.adjustments.map(&:amount)).to match_array([0.01, -0.83, -0.84])
```

Comment:
`[0.01, -0.83, -0.84].inject(:+) #=> -1.66`

and that 0.01 is the tax adjustment above :+1: